src/test/java/page/GoogleLoginPage.java

```diff
@@ -34,7 +35,9 @@ public GoogleMailHomePage loginToGoogleMail(User user) {
         confirmEmailOrPhoneButton.click();
         log.info("Login entered");
         wait.until(ExpectedConditions.visibilityOf(passwordTextField));
-        passwordTextField.sendKeys(user.getPassword());
+        if (user.getPassword() != null) {
```

Comment:
Should we get null here? I think it's a chance to implement a true get/set method